File: 프로그래머스/lv0/120846. 합성수 찾기/합성수 찾기.py
def solution(n):
    answer = 0
    for i in range(1,n+1):        
        lst=[]
        for j in range(1,i+1):
            if i % j ==0:
                lst.append(j)
        if len(lst) >= 3:
            answer+=1      
    return answer

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        for(int i=4; i<=n; i++){
            int tmp = 1;
            for(int j=2; j<=i; j++){
                if(i%j==0){
                    tmp+=1;
                }
            }
            if(tmp >=3){
                answer+=1;
            }
        }
        return answer;
    }
}